phpcs.xml.dist

```diff
@@ -47,17 +47,18 @@
         <exclude-pattern>lib/Doctrine/Migrations/AbstractMigration.php</exclude-pattern>
         <exclude-pattern>tests/Doctrine/Migrations/Tests/Configuration/AbstractConfigurationTest.php</exclude-pattern>
         <exclude-pattern>tests/Doctrine/Migrations/Tests/Tools/Console/Command/AbstractCommandTest.php</exclude-pattern>
+        <exclude-pattern>tests/Doctrine/Migrations/Tests/Configuration/Loader/AbstractLoaderTest.php</exclude-pattern>
+        <exclude-pattern>lib/Doctrine/Migrations/Configuration/Loader/AbstractFileLoader.php</exclude-pattern>
     </rule>
 
     <rule ref="SlevomatCodingStandard.Classes.SuperfluousInterfaceNaming">
         <exclude-pattern>lib/Doctrine/Migrations/Configuration/Connection/ConnectionLoaderInterface.php</exclude-pattern>
-        <exclude-pattern>lib/Doctrine/Migrations/Provider/SchemaDiffProviderInterface.php</exclude-pattern>
-        <exclude-pattern>lib/Doctrine/Migrations/Provider/SchemaProviderInterface.php</exclude-pattern>
         <exclude-pattern>lib/Doctrine/Migrations/Tools/Console/Helper/ConfigurationHelperInterface.php</exclude-pattern>
         <exclude-pattern>lib/Doctrine/Migrations/Finder/MigrationFinderInterface.php</exclude-pattern>
-        <exclude-pattern>lib/Doctrine/Migrations/Finder/MigrationDeepFinderInterface.php</exclude-pattern>
         <exclude-pattern>lib/Doctrine/Migrations/Version/ExecutorInterface.php</exclude-pattern>
         <exclude-pattern>lib/Doctrine/Migrations/ParameterFormatterInterface.php</exclude-pattern>
-        <exclude-pattern>lib/Doctrine/Migrations/Version/VersionInterface.php</exclude-pattern>
+        <exclude-pattern>lib/Doctrine/Migrations/MigratorInterface.php</exclude-pattern>
```

Comment:
Since @lcobucci brought it up in his review, I'd appreciate it if we can revisit interface names here.

Reply:
Ok, will do